test/test_bored.rb

```diff
@@ -21,4 +21,19 @@ def test_that_it_gives_you_an_activity
     assert (0..1).cover?(activity.price)
     assert [String, NilClass].include?(activity.link.class)
   end
+
+  def test_that_you_can_query_by_participants
+    activity = Bored.now(participants: 3)
+
+    assert_kind_of Numeric, activity.id
+    assert_kind_of String, activity.description
+    assert_includes([
+      :education, :recreational, :social, :diy, :charity,
+      :cooking, :relaxation, :music, :busywork
+    ], activity.type)
+    assert_equal 3, activity.participants
+    assert (0..1).cover?(activity.accessibility)
+    assert (0..1).cover?(activity.price)
+    assert [String, NilClass].include?(activity.link.class)
+  end
```

Comment:
Given that we already have a test for the presence of the other fields, I think it'd probably be sufficient to only assert that the `activity.participants` is 3. Does that sound ok to you?

Reply:
Sorry -- I guess I didn't have notifications on either! That sounds fine to me, going forward at least (since this is already merged!).